integration_tests/conftest.py

```diff
@@ -16,16 +16,23 @@ def sim() -> InteractiveContext:
     return simulation
 
 
-TIME_STEPS_TO_TEST = [0, 1, 10]
+TIME_STEPS_TO_TEST = [0, 1]
 
 
 @pytest.fixture(scope="session")
 def populations(sim) -> List[pd.DataFrame]:
     population_states = []
     for _ in range(max(TIME_STEPS_TO_TEST) + 1):
         pop = sim.get_population(untracked=True)
+        pipelines = sim.list_values()
+        for pipeline in pipelines:
+            # Metrics is a dict we cannot concat and do not want to
+            if pipeline == "metrics":
+                continue
+            pop = pd.concat([pop, sim.get_value(pipeline)(pop.index)], axis=1)
+            # Handle column names for pipelines
+            pop = pop.set_axis([*pop.columns[:-1], pipeline], axis=1, inplace=False)
```

Comment:
What is this doing? Is `pipeline` always a string? Couldn't it just be:

`pop[pipeline] = sim.get_value(pipeline)(pop.index)`

Reply:
The pipeline wasn't always named so there were columns with no names in the dataframe.  This is giving the columns the name of the pipeline.

Reply:
Right, but if the `pipeline` variable is always a string name for the column, why wouldn't my suggestion work to replace lines 32-34?

Reply:
Oh you are right.  Thanks.

Reply:
So this doesnt work because one of the pipelines seems to be a dataframe which doesnt allow it to be a column.